### ISSUE TITLE
Color Contrast on New Email Icon Doesn't Pass WCAG Guidelines

### ISSUE THREAD

**Issue by bobmatyas** [tutao/tutanota#1542]
- [x] This is not a feature request. I will request features on the forum or via support (see [contact page](https://tutanota.com/contact?page=feature)).
- [x] This issue doesn't fit into provided templates.

In dark mode, the color contrast on the circular "new email" button fails according to the WCAG Guidelines. 

The failing combo is: 

Foreground: #FFFFFF; 
Background: #76CBDA;

For a ratio of 1.85:1  -- https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=76CBDA

A passing ratio should be at least 4.5:1. This could be achieved by swapping the white color (#FFFFFF) for a grey used elsewhere in the UI with the blue background (for example on the labels indicating an email came from the Tutanota Team).

That combo would be:

Foreground: #4a4a4a;
Background: #76CBDA;

For a ratio of 4.77:1 -- https://webaim.org/resources/contrastchecker/?fcolor=4A4A4A&bcolor=76CBDA

I'd be happy to do a PR for this fix makes sense to you all.

**Comment by charlag** [comment]
Hi, thanks for reporting
We plan to change dark theme in any case, we will keep this in mind when doing it.

**Comment by bobmatyas** [comment]
Excellent, I'm excited to see the new dark theme!

**Comment by charlag** [comment]
Fixed by ff61e50dbe5b8336f78de65692c0852f129f6916
